.github/workflows/main.yml

```diff
@@ -61,6 +61,13 @@ jobs:
 
       - name: Run ${{ matrix.command }} task
         run: npm run ${{ matrix.command }}
+        env:
+          CI_REPO_OWNER: Leaflet
+          CI_REPO_NAME: Leaflet
+          CI_COMMIT_MESSAGE: N/A
+          CI_COMMIT_SHA: ${{ github.sha }}
+          # BUNDLESIZE_GITHUB_TOKEN 
```

Comment:
What is the purpose of the commit related environment variables? Are we expecting that this tool will be making commits on the repo?

Reply:
I'm not shure with CI_COMMIT_MESSAGE but as this is CI undependent it has to get the information where to push the status. Therefore it needs Owner, Name and the SHA for wich commit to attach the status. Commits to the repo would be bad.

Reply:
And I checked the link to generate the token only allowes write access to the status not any commit rights.